internal/cmd/pipeline/command_update.go

```diff
@@ -44,6 +53,7 @@ func (c *command) newUpdateCommand(enableSourceCode bool) *cobra.Command {
 			"If <secret-value> is empty, the named secret will be removed from Stream Designer.")
 	}
 	cmd.Flags().Bool("activation-privilege", true, "Grant or revoke the privilege to activate this pipeline.")
+	cmd.Flags().Bool("update-schema-registry", true, "Update the pipeline with the latest schema registry cluster.")
```

Comment:
Is the use case that customers want to be able to update their pipeline without updating their Schema Registry cluster? Not sure why we'd want that. Maybe you meant the default to be `false` here?